### PR TITLE
Update Github Substrate Master Links

### PR DESCRIPTION
**Pull request by shawntabrizi**
This PR will attempt to only rename links

### CODE REVIEW

File: docs/conceptual/runtime/contracts/contracts_module.md [hunk 1]
```diff
@@ -107,18 +107,17 @@ blockchain.
 ### Learn More
 
 - Learn about [ink!, a Rust based embedded domain specific
-  language](conceptual/runtime/contracts/ink.md) for developing smart contracts for the SRML
+  language](conceptual/runtime/contracts/ink.md) for developing smart contracts for Substrate Pallete
```
Comment (folsen) [inline docs/conceptual/runtime/contracts/contracts_module.md]: Should be "for the contracts pallet" for consistency with the other stuff above no?

Comment (shawntabrizi) [reply docs/conceptual/runtime/contracts/contracts_module.md]: Yes, this was missed by the find replace because of newline :/

**Comment by shawntabrizi** [comment]
This should be ready for review. Updates all links with some "github master" link to hopefully the appropriate place.

File: docs/conceptual/runtime/srml.md [hunk 2]
```diff
@@ -89,199 +89,199 @@ authorities, learn its own authority id, as well as to sign and verify messages
...
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/generic-asset/src/lib.rs)
 
 ### GRANDPA
 
 The GRANDPA module extends GRANDPA consensus by managing the GRANDPA authority set ready for the
 native code.
 
 * [Docs](/rustdocs/master/srml_grandpa/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/grandpa/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/grandpa/src/lib.rs)
```
Comment (riusricardo) [inline docs/conceptual/runtime/srml.md]:
```suggestion
* [Source](https://github.com/paritytech/substrate/blob/master/palette/grandpa/src/lib.rs)
```
When a module belongs to the core is `palette` not `pallete`.

File: docs/conceptual/runtime/srml.md [hunk 3]
```diff
@@ -89,199 +89,199 @@ authorities, learn its own authority id, as well as to sign and verify messages
...
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/membership/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/membership/src/lib.rs)
 
 ### Offences
 
 The Offences module tracks reported offences.
 
 * [Docs](/rustdocs/master/srml_offences/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/srml/offences/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/pallete/offences/src/lib.rs)
```
Comment (riusricardo) [inline docs/conceptual/runtime/srml.md]:
```suggestion
* [Source](https://github.com/paritytech/substrate/blob/master/palette/offences/src/lib.rs)
```

File: docs/conceptual/runtime/srml.md (hunk 2 above)
Comment (shawntabrizi) [reply docs/conceptual/runtime/srml.md]: ah, sorry, probably many of these errors to come

Comment (shawntabrizi) [reply docs/conceptual/runtime/srml.md]: updated

Comment (riusricardo) [reply docs/conceptual/runtime/srml.md]: So I think that replacing should work but it need to be `palette`
The differentiation is on the package name.

Comment (riusricardo) [reply docs/conceptual/runtime/srml.md]: Ex.
`pallet-balances`
`pallet-contracts`

Comment (riusricardo) [reply docs/conceptual/runtime/srml.md]: `pallet-contracts`
`palette-support`

**Pull request merged**